packages/lodestar/src/network/peers/utils.ts

```diff
@@ -50,8 +61,8 @@ export async function handlePeerMetadataSequence(
 /**
  * Find subnets that we don't have at least 1 connected peer.
  */
-export function findMissingSubnets(network: INetwork): number[] {
-  const peers = network.getPeers({connected: true}).map((peer) => peer.id);
+export function findMissingSubnets(connectedPeers: LibP2p.Peer[] = [], network: INetwork): number[] {
```

Comment:
Can we use type `PeerId` for `connectedPeers` param here and below?
Also, connectedPeers shouldn't have default value of `[]`

Reply:
yeah I changed to `PeerId[]` for possible apis. Some still need to access the internal `protocols` so we have to go with `Libp2p.Peer[]`.
also I removing the default `[]` from all of them